test/webgl/context-state.spec.js

```diff
@@ -151,3 +151,90 @@ test('WebGLState#withParameters', t => {
 
```

Comment:
`setParameters` and `withParameters` are defined in context-state.js, so keeping the same name, context-state.spec.js, any other specific reason you want to change it?